client/my-sites/upgrades/controller.jsx

```diff
@@ -161,6 +171,50 @@ module.exports = {
 		);
 	},
 
+	selectPlan: function( context ) {
+		const basePath = route.sectionify( context.path );
+		const CartData = require( 'components/data/cart' );
+		const Plans = require( './plans' );
+		const domain = context.params.registerDomain;
+
+		const onGoBack = () => {
+			page( '/domains/add/' + sites.getSelectedSite().slug );
+		};
+
+		analytics.pageView.record( basePath, 'Domain Search > Domain Registration > Select Plan' );
```

Comment:
I realize that this is the controller fix, but could we use the `<PageViewTracker />` component to keep away from directly calling `analytics.pageView`?

Alternatively we could fire off the action directly from the store…

```js
import { recordPageView } from 'state/analytics/actions';

context.store.dispatch( recordPageView( basePath, 'Domain Search…' ) );
```

obviously the `<PageViewTracker />` has its advantages because we can test and all without ever calling to the analytics code